problem1.py

```diff
@@ -0,0 +1,21 @@
+#timecomplexity : can you help to find the complexity of this code for iterative and recursion both
```

Comment:
The time and space here will be of an Inorder traversal.
O(n) time, each node visited once; O(h) Space